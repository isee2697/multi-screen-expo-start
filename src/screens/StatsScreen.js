import React from 'react';
import { ScrollView, StyleSheet, Text, View } from 'react-native';
import { colors } from '../api/constants';

const StatsScreen = () => (
  <View style={styles.container}>
    <ScrollView
      style={styles.container}
      contentContainerStyle={styles.contentContainer}
    >
      <Text style={styles.textStyle}>Stats Content Area</Text>
    </ScrollView>
  </View>
);

StatsScreen.navigationOptions = {
  headerTitleStyle: {
    flex: 1,
    textAlign: 'center'
  },
  title: 'Stats'
};

const styles = StyleSheet.create({
  container: {
    backgroundColor: colors.brandPrimary,
    flex: 1
  },
  contentContainer: {
    alignItems: 'center',
    paddingTop: 30
  },
  textStyle: {
    color: colors.white
  }
});

export default StatsScreen;
