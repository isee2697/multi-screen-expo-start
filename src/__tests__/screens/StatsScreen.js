import React from 'react';
import { shallow } from 'enzyme';

import StatsScreen from '../../screens/StatsScreen';

const navigation = { navigate: jest.fn() };
const screen = shallow(<StatsScreen navigation={navigation} />);

describe('<StatsScreen />', () => {
  it('renders correctly', () => {
    expect(screen).toMatchSnapshot();
  });
});
