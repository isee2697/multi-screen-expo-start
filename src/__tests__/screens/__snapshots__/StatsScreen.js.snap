// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StatsScreen /> renders correctly 1`] = `
<Component
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
>
  <ScrollViewMock
    contentContainerStyle={
      Object {
        "alignItems": "center",
        "paddingTop": 30,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#1d2d3a",
        }
      }
    >
      Stats Content Area
    </Text>
  </ScrollViewMock>
</Component>
`;
